XIVSlothCombo/Combos/DRG.cs

```diff
@@ -382,31 +431,56 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     }
                 }
 
-                //Dives under Litany Feature
+                //Dives under Litany and Life of the Dragon Feature
                 if (canWeaveAbilities)
                 {
 
-                    if (IsEnabled(CustomComboPreset.DragoonLitanyDiveFeature))
+                    if (IsEnabled(CustomComboPreset.DragoonLifeLitanyDiveFeature))
                     {
-                        var gauge = GetJobGauge<DRGGauge>();
                         if (
                             gauge.IsLOTDActive == true &&
                             level >= DRG.Levels.DragonfireDive &&
                             HasEffect(DRG.Buffs.BattleLitany) &&
-                            IsOffCooldown(DRG.DragonfireDive) && canWeaveAbilities
+                            IsOffCooldown(DRG.DragonfireDive) && CanWeave(actionID, weaveTime: 0.9)
```

Comment:
You can pass things like weavetime just by specifying a number e.g. (actionID, 0.7)